api/main_endpoints/routes/User.js

```diff
@@ -305,4 +305,25 @@ router.post('/connectToDiscord', function(req, res) {
     );
 });
 
+router.post('/getUserById', async (req, res) => {
```

Comment:
need to add token checking here otherwise anyone can look users up 
```js
  if (!checkIfTokenSent(req)) {
    return res.sendStatus(FORBIDDEN);
  } else if (!checkIfTokenValid(req, (
    membershipState.OFFICER
  ))) {
    return res.sendStatus(UNAUTHORIZED);
  }
```

accounting for this in ur tests should be easy